examples/rosettacode/100_doors.nit

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env nit
+#
+# This file is part of NIT ( http://www.nitlanguage.org ).
+# This program is public domain
+
+
+
+class Door
+	#True : Open / False : Close
+	var state: Bool = false
+
+	fun is_open: Bool
+	do
```

Comment:
For such small functions you can inline them
`fun is_open: Bool do return state`

The self can be implicit too